dev-support/docker/Dockerfile

```diff
@@ -91,38 +95,20 @@ ENV MAVEN_HOME /usr
 # JAVA_HOME must be set in Maven >= 3.5.0 (MNG-6003)
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 ENV FINDBUGS_HOME /usr
+ENV PROTOBUF_HOME /opt/protobuf
+ENV PATH "${PATH}:/opt/protobuf/bin"
 
-#######
-# Install Boost 1.72 (1.65 ships with Bionic)
-#######
-# hadolint ignore=DL3003
-RUN mkdir -p /opt/boost-library \
-    && curl -L https://sourceforge.net/projects/boost/files/boost/1.72.0/boost_1_72_0.tar.bz2/download > boost_1_72_0.tar.bz2 \
-    && mv boost_1_72_0.tar.bz2 /opt/boost-library \
-    && cd /opt/boost-library \
-    && tar --bzip2 -xf boost_1_72_0.tar.bz2 \
-    && cd /opt/boost-library/boost_1_72_0 \
-    && ./bootstrap.sh --prefix=/usr/ \
-    && ./b2 --without-python install \
-    && cd /root \
-    && rm -rf /opt/boost-library
-
-######
-# Install Google Protobuf 3.7.1 (3.0.0 ships with Bionic)
-######
+####
+# Install pip (deprecated from Focal toolchain)
+####
 # hadolint ignore=DL3003
-RUN mkdir -p /opt/protobuf-src \
-    && curl -L -s -S \
-      https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protobuf-java-3.7.1.tar.gz \
-      -o /opt/protobuf.tar.gz \
-    && tar xzf /opt/protobuf.tar.gz --strip-components 1 -C /opt/protobuf-src \
-    && cd /opt/protobuf-src \
-    && ./configure --prefix=/opt/protobuf \
-    && make install \
+RUN mkdir -p /opt/pip \
```

Comment:
Why not use pip from the repo?

Reply:
As mentioned in this stackoverflow post - https://askubuntu.com/questions/1242684/python-pip-package-disappeared-in-ubuntu-20-04, pip is deprecated in Focal and thus, we need to install is separately

Also, I couldn't find python2-pip in the Focal packages - https://packages.ubuntu.com/focal/allpackages?format=txt.gz

Reply:
OK, eventually we need to upgrade to python3 or so, but for now let's go for this.